src/site/resources/css/site.css

```diff
@@ -163,4 +163,8 @@ a[title="toTop"] {
     display: block;
     text-align: left;
   }
+
+  section[id$="_Properties"] .wrapper .bodyTable tr td:first-child {
```

Comment:
Looks like there is some issue in new pages if the ID starts with `_`. Originally IDs were like `id="AnnotationLocation_Properties"` before we split. The `_` was added to provide a split between check name and section name. The `_` was not removed when it should have been.

Reply:
Yes, better to be without leading `_`

Reply:
Done.

Reply:
@stoyanK7 I don't see any other code changes but removing the `_`. Is this a `find` and not a complete `match`?

Looking at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/b00c60f_2023120428/checks/annotation/annotationlocation.html#AnnotationLocation_Description
It seems we still have the same pattern `check name _ section name`. I don't see the point in duplicating the check name with the same page name.
These `href`s should be changed for the new page style.

Reply:
This should be a complete match when the `href`s are fixed.

Reply:
Mentioned in issue.